docs/charts/doughnut.md

```diff
@@ -71,6 +84,18 @@ The following values are supported for `borderAlign`.
 
 When `'center'` is set, the borders of arcs next to each other will overlap. When `'inner'` is set, it is guaranteed that all the borders are not overlap.
 
+### Interactions
+
+The interaction with each arc can be controlled with the following properties:
+
+| Name | Description
+| ---- | -----------
+| `hoverBackgroundColor` | arc background color when hovered.
+| `hoverBorderColor` | arc border color hovered.
```

Comment:
arc border color when hovered

Reply:
Done. I noticed this same error was in the bubble & bar chart docs too so I've updated it there as well.